src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -115,6 +119,10 @@ abstract contract DelegationManagerStorage is IDelegationManager {
     /// @dev This variable only reflects withdrawals that were made after the slashing release.
     mapping(bytes32 withdrawalRoot => Withdrawal withdrawal) public queuedWithdrawals;
 
+    /// @dev Contains history of the total cumulative staker withdrawals for a given strategy. Used to calculate
+    /// burned shares for StrategyManager shares.
+    mapping(IStrategy strategy => Snapshots.WithdrawalHistory) internal _cumulativeWithdrawalsHistory;
```

Comment:
This needs to be storage on a per operator, strategy basis? Otherwise we burn all queued withdrawals that were slashed for a given strategy regardless of operator?

Reply:
Just fixed, realized this miss too